### PR TITLE
use debian:stable-slim as base

### PR DESCRIPTION
**Pull request by dulmandakh**
# Summary

Currently, we use openjdk:8-slim as base. But also we install openjdk-8-jre due to issues. So why don't we base off debian:stable-slim and install openjdk-8-jdk from repository, instead of basing off openjdk image and installing duplicate jre (openjdk-8-jre). which increases image size.

Also creating /usr/share/man/man1 in single step with package installation to reduce image layers.

## Test Plan

Everything should work just fine.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by dulmandakh** [comment]
@gengjiawen why is CI not running?

**Comment by gengjiawen** [comment]
> @gengjiawen why is CI not running?

Looks like CircleCI bug.

File: Dockerfile [hunk 1]
```diff
@@ -8,37 +8,35 @@ ARG ANDROID_BUILD_VERSION=28
...
+RUN mkdir -p /usr/share/man/man1 \
+    && apt-get update -qq && apt-get install -qq -y --no-install-recommends \
         apt-transport-https \
         curl \
         build-essential \
         file \
         git \
         gnupg2 \
-        openjdk-8-jre \
+        openjdk-8-jdk \
```
Comment (gengjiawen) [inline Dockerfile]: We may still need jre, last time I remember emulator depend on this.

Comment (dulmandakh) [reply Dockerfile]: openjdk-8-jdk depends on openjdk-8-jre

**Comment by dulmandakh** [comment]
@gengjiawen what should I do? Yesterday, I played with NDK 19, while I was setting up my laptop, and it worked just fine

File: Dockerfile (hunk 1 above)
Comment (gengjiawen) [reply Dockerfile]: That's what I think, but from result we still need it. Let's see the CI result.

**Comment by gengjiawen** [comment]
> @gengjiawen what should I do? Yesterday, I played with NDK 19, while I was setting up my laptop, and it worked just fine. So I'll bump NDK once this merged.

Add the Ndk bump commit and let's see the CI will trigger ?

**Pull request merged**